lib/MetaCPAN/Script/Release.pm

```diff
@@ -466,7 +404,7 @@ sub import_tarball {
         }
     }
     if (@provides) {
-        $release->provides( [ sort @provides ] );
+        $release->provides( \@provides );
```

Comment:
In the new version I did not, this why I said I messed up with git, I solved some merge conflicts and deleted some things that shouldn't have been deleted. The sort stays.